dashboard/app/models/script_level.rb

```diff
@@ -67,6 +67,16 @@ def end_of_stage?
     stage.script_levels.to_a.last == self
   end
 
+  def long_assessment?
+    if self[:assessment]
+      level_info = Level.find(self[:level_id])
+      if level_info[:properties]["pages"] && level_info[:properties]["pages"].length > 1
```

Comment:
does `level.pages` work? SerializedAttributes should write helper methods for stuff in properties.